### PR TITLE
Feat: Added confirmation to create new file

### PR DESCRIPTION
**Pull request by ashmit-coder**
**Description**

- The feature creates a modal prompt before creating a new file to prevent user from loosing his data

- Old Behaviour

https://github.com/asyncapi/studio/assets/69006513/d7945641-2319-4de4-af88-b4755da8762b

- Current Behaviour

https://github.com/asyncapi/studio/assets/69006513/8529183a-3d15-440d-9124-d885994d84ec

**Related issue(s)**

Resolves: #724

### CODE REVIEW

File: apps/studio/src/components/Modals/ConfirmNewFileModal.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      containerClassName="sm:max-w-6xl"
+      title="Confirm New File"
+      confirmText="Create New File"
+      confirmDisabled={false}
+      onSubmit={onConfirm}
+    >
+      <div className="flex content-center justify-center flex-col">
+        <div className="w-full  overflow-auto">
+          <div>
+                        Do you want to create New File?
```
Comment (Amzani) [inline apps/studio/src/components/Modals/ConfirmNewFileModal.tsx:22]:
```suggestion
                        Would you like to create a new file?
```

**Comment by ashmit-coder** [comment]
Hey @Amzani Please check the Updates. :)

**Review by Amzani — APPROVED** [review]
LGTM

**Comment by ashmit-coder** [comment]
Hey @Amzani I guess i did something wrong but the PR wont merge. Appologies from my side.

**Pull request merged**